src/tests/ftest/scripts/main.sh

```diff
@@ -190,6 +190,20 @@ fi
 
 export DAOS_APP_DIR=${DAOS_APP_DIR:-$DAOS_TEST_SHARED_DIR}
 
+# Create two test users in the same group
+if ! ./user_setup.py -n "${TEST_NODES}" -g daos_test_groupx -u daos_test_userx1,daos_test_userx2 -s; then
```

Comment:
Optional readability improvement:
```suggestion
if ! ./user_setup.py -n "${TEST_NODES}" -g daos_test_group_x -u daos_test_user_x1,daos_test_user_x2 -s; then
```

Reply:
Agree. Thanks